### ISSUE TITLE
Toggle fields are not interactable via keyboard

### ISSUE THREAD

**Issue by d4mation** [realbig/rbm-field-helpers#104]
This is an accessibility issue. You cannot tab into the field or interact with it in any way. You'll just end up skipping over it.
